### ISSUE TITLE
Yolov4 iou loss remain 0 while training

### ISSUE THREAD

**Issue by kaijaz123** [pjreddie/darknet#2431]
Here is the setting for .cfg file:

# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 6000
policy=steps
steps=4800,5200
scales=.1,.1

#cutmix=1
mosaic=1](url)

Score:
185: 0.237495, 0.974284 avg loss, 0.000001 rate, 3.276501 seconds, 11840 images, 7.041305 hours left
Loaded: 0.539213 seconds - performance bottleneck on CPU or Disk HDD/SSD
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 139 Avg (IOU: 0.000000), count: 1, class_loss = 0.192889, iou_loss = 0.000000, total_loss = 0.192889

Did anyone has the same issue while training?
